frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentAccordionBody/StrategyDraggableItem/StrategyItem/CopyStrategyIconMenu/CopyStrategyIconMenu.tsx

```diff
@@ -82,14 +82,14 @@ export const CopyStrategyIconMenu: VFC<ICopyStrategyIconMenuProps> = ({
             await addStrategyToFeature(
                 projectId,
                 featureId,
-                environmentId,
+                environment,
```

Comment:
This PR seems to be only about changing name of parameter? Am I misunderstanding something?

Reply:
`EnvironmentId` is the src environment. `environment` is passed in the function as the target